liam2/exprmisc.py

```diff
@@ -160,6 +165,21 @@ def compute(self, context, expr):
 
     dtype = always(int)
 
+
+class Erf(FunctionExpr):
+    def compute(self, context, expr):
+        if scipy is None:
+            raise ImportError(
+                "Scipy was not succesfully imported",

```

Comment:
I would like a better message for people trying to use it without scipy installed. Something like: 
"Failed to import scipy, which is required for erf(). Please make sure scipy is installed and working."